Gruntfile.js

```diff
@@ -1,47 +1,67 @@
 module.exports = function(grunt) {
-  grunt.loadNpmTasks('grunt-contrib-concat');
-  grunt.loadNpmTasks('grunt-contrib-uglify');
-  grunt.loadNpmTasks('grunt-karma');
 
-  var sourceFiles = grunt.file.readJSON('build.json');
-  grunt.initConfig({
-    concat: {
-      pointerevents: {
-        options: {
-          stripBanners: true,
-          banner: grunt.file.read('LICENSE.txt')
-        },
-        nonull: true,
-        src: sourceFiles,
-        dest: 'pointerevents.dev.js'
-      }
-    },
-    uglify: {
-      pointerevents: {
-        options: {
-          // sourceMap: 'pointerevents.min.js.map',
-          banner: grunt.file.read('LICENSE.txt')
-        },
-        nonull: true,
-        dest: 'pointerevents.min.js',
-        src: sourceFiles
-      }
-    },
-    karma: {
+grunt.loadNpmTasks('grunt-contrib-uglify');
+grunt.loadNpmTasks('grunt-karma');
+
+var package = require('./package');
```

Comment:
Just remembered: Based on using `throws` in QUnit, I'd recommend avoiding it, since there's always some shitty tool that doesn't deal with ex-reserved words properly.